stl/inc/regex

```diff
@@ -1202,7 +1202,7 @@ enum _Node_flags : int { // flags for nfa nodes with special properties
     _Fl_none    = 0x00,
     _Fl_negate  = 0x01,
     _Fl_greedy  = 0x02,
-    _Fl_final   = 0x04,
+    _Fl_final   = 0x04, // TRANSITION, ABI: parser used to mark some nodes with this flag

```

Comment:
We can remove the `_Fl_final` name without disturbing the other enumerators, and adjust the comment accordingly.